bindings/matrix-sdk-crypto-js/tests/helper.js

```diff
@@ -0,0 +1,80 @@
+const { DeviceLists, RequestType, KeysUploadRequest, KeysQueryRequest } = require('../pkg/matrix_sdk_crypto_js');
+
+function* zip(...arrays) {
```

Comment:
what's that `function*` notation? What does that mean? Not seen that in JS before!?!

Reply:
It's used to declare a [`Generator`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Generator), with `yield` to produce a value.